meson.build

```diff
@@ -1,29 +1,37 @@
-project('tinyendian', 'd',
-    meson_version: '>=0.40',
-    version: '0.2.0'
-)
+# -*- mode: python; -*-
 
-project_soversion    = '0'
+project(
+    'tinyendian',
+    'd',
+    meson_version: '>=0.41',
+    version: '0.2.0',
+    default_options: ['buildtype=release'],
```

Comment:
I am not sure if changing the default is wise here - it's quite unexpected to have one project build differently, especially in a mode that doesn't include debugging symbols.